crates/edr_eth/src/remote/request_methods.rs

```diff
@@ -100,6 +100,7 @@ pub enum RequestMethod {
 }
 
 impl RequestMethod {
```

Comment:
Is this only used when tracing is enabled?

Reply:
Yes. I added this because otherwise `cargo hack` would complain when running clippy without the tracing feature.

Is there a better way to handle this?

Reply:
Nope, just validating